src/main/java/org/skytest/questiontest/controller/JavaQuestionController.java

```diff
@@ -0,0 +1,39 @@
+package org.skytest.questiontest.controller;
+
+import org.skytest.questiontest.model.Question;
+import org.skytest.questiontest.service.JavaQuestionService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam/java")
+public class JavaQuestionController {
+    private final JavaQuestionService service;
+
+    public JavaQuestionController(JavaQuestionService service) {
```

Comment:
используем полиморфизм

```suggestion
    public JavaQuestionController(QuestionService service) {
```